javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -211,6 +215,13 @@ private TreeUtils() {
             err.initCause(e);
             throw err;
         }
+
+        BINARY_COMPARISON_TREE_KINDS.add(Tree.Kind.EQUAL_TO);
```

Comment:
If you use `EnumSet`, using `EnumSet.of(...)` might be a bit shorter.

Reply:
Thanks!